openlibrary/solr/updater/edition.py

```diff
@@ -158,6 +158,13 @@ def number_of_pages(self) -> int | None:
         except (TypeError, ValueError):  # int(None) -> TypeErr, int("vii") -> ValueErr
             return None
 
+    @property
+    def translation_of(self) -> str | None:
+        try:
+            return self._edition.get("translation_of", None) or None
+        except (TypeError, ValueError):
+            return None
```

Comment:
This will never throw an error so we don't need a try/except. The above block needed the try/except because of the `int` call.

```suggestion
        return self._edition.get("translation_of")
```